oauth2_http/java/com/google/auth/oauth2/OAuth2Utils.java

```diff
@@ -17,7 +19,7 @@
  * Internal utilities for the com.google.auth.oauth2 namespace.
  */
 class OAuth2Utils {
-  static final String TOKEN_SERVER_URL = "https://accounts.google.com/o/oauth2/token";
+  static final URI TOKEN_SERVER_URI = parseStaticUri("https://accounts.google.com/o/oauth2/token");
```

Comment:
Yes, this is intentional as you can't initialize a static field with something that can throw a checked exception type. From checking Stack Overflow, this seems to be how most people work around the problem.